recipes/libgpuarray/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+if [ "$(uname)" == "Darwin" ]; then
+  export MACOSX_VERSION_MIN=10.7
+  export CC=clang
+  export CXX=clang++
+  export CFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+fi
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX
+cmake --build . --config Release --target all
```

Comment:
The test step would require CUDA or OpenCL on the build machines, which is why I avoided it.